Makefile

```diff
@@ -17,7 +17,7 @@ REPO_PATH="github.com/kubeflow/mpi-operator"
 REL_OSARCH="linux/amd64"
 GitSHA=$(shell git rev-parse HEAD)
 Date=$(shell date "+%Y-%m-%d %H:%M:%S")
-RELEASE_VERSION?=v0.3.0
+RELEASE_VERSION?=v0.4.0
```

Comment:
actually, we used `0.3.0` in the tag before, as opposed to `v0.3.0`

Reply:
Upon confirmation, only `v0.2.3` has a `v` prefix.